src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -574,7 +575,7 @@ void SpatialPooler::initialize(vector<UInt> inputDimensions,
 
   tieBreaker_.resize(numColumns_);
   for (UInt i = 0; i < numColumns_; i++) {
-    tieBreaker_[i] = 0.01 * rng_.getReal64();
+    tieBreaker_[i] = pow(10, -1*(getPrecision()+1)) * rng_.getReal64(); // (PRECISION+1)-th place
```

Comment:
let's move the discussion to a separate issue here: https://github.com/numenta/nupic/issues/2629